### PR TITLE
Tx Pool improvements

### PR DESCRIPTION
**Pull request by andreibancioiu**
https://github.com/ElrondNetwork/elrond-go/blob/pool++/storage/txcache/README.md

### CODE REVIEW

File: storage/txcache/eviction.go [hunk 1]
```diff
@@ -79,6 +79,19 @@ func (cache *TxCache) shouldContinueEvictingSenders() bool {
 	return cache.areThereTooManyTxs() || cache.areThereTooManySenders() || cache.areThereTooManyBytes()
 }
 
+func (cache *TxCache) isHighLoad() bool {
+	// TODO: Improve condition
+	const highLoadPerCapacity = 0.5
```
Comment (sasurobert) [inline storage/txcache/eviction.go:84]: move const to top of file

**Comment by elrondbigscam** [comment]
ERD scam coin, don't buy.  The ERD team is a crook.

File: storage/txcache/eviction.go (hunk 1 above)
Comment (elrondscammer2) [reply storage/txcache/eviction.go]: Elrond was Hacked. Sell now

[the remark this answers is not in the document]
Comment (elronscam8) [reply storage/txcache/eviction.go]: 25 sats | dump dump dump dump dump |Shit boys, you have no success. Swindling bastards

Comment (elronscam8) [reply storage/txcache/eviction.go]: 25 sats | dump dump dump dump dump |Shit boys, you have no success. Swindling bastards

**Pull request closed without merging**